src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -130,14 +130,14 @@ public GetBalanceResponse getBalance(final String accountId, final String bearer
     }
 
     @Override
-    public SendPaymentResponse sendPayment(String destinationPaymentPointer,
-                                           UnsignedLong amount,
-                                           String accountId,
-                                           String bearerToken) throws XpringException {
+    public SendPaymentResponse sendPayment(final UnsignedLong amount,
```

Comment:
This is fine for now, but will probably lead to errors. For future, we should consider a value object here so that it's obvious which values are getting set.

Reply:
Agreed.  @keefertaylor and I decided we should wrap protobuf objects so as not to expose them in the public API.  Once we do this we can consolidate these parameters into one of those wrapped objects.

See https://app.asana.com/0/1143722888069470/1165556580466065